### PR TITLE
:bug:  fix side-nav collapse/pin icon color

### LINKED ISSUE #1416: [Bug]: collapse sidenav icon color
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

The collapse side-nav icon color used to be white. Now it's black and barely visible

Screenshot 2023-07-29 at 12 01 03

### What error did you receive?

_No response_

### Where are you hosting Actual?

None

### What browsers are you seeing the problem on?

_No response_

### Operating System

None

### PR DESCRIPTION
**Pull request by MatissJanis**
Closes #1416

### CODE REVIEW

File: packages/desktop-client/src/components/sidebar.js [hunk 1]
```diff
@@ -445,10 +445,7 @@ function Accounts({
 
 function ToggleButton({ style, isFloating, onFloat }) {
   return (
-    <View
-      className="float"
-      style={[style, { flexShrink: 0, color: colors.n5 }]}
-    >
+    <View className="float" style={[style, { flexShrink: 0 }]}>
       <Button type="bare" onClick={onFloat}>
```
Comment (j-f1) [inline packages/desktop-client/src/components/sidebar.js:449]: This should have the same effect:
```suggestion
    <View className="float" style={[style, { flexShrink: 0 }]}>
      <Button type="bare" onClick={onFloat} style={{ color: colors.n5 }}>
```

…without having to duplicate the color name

**Review by j-f1 — APPROVED** [review]

File: packages/desktop-client/src/components/sidebar.js (hunk 1 above)
Comment (MatissJanis) [reply packages/desktop-client/src/components/sidebar.js]: Good tip! https://github.com/actualbudget/actual/pull/1421/commits/cd593440eeab2a40db03daa5754a4f92638382be

**Review by j-f1 — DISMISSED** [review]

**Comment by j-f1** [comment]
agh never mind

File: packages/desktop-client/src/components/sidebar.js [hunk 2]
```diff
@@ -445,11 +445,8 @@ function Accounts({
 
 function ToggleButton({ style, isFloating, onFloat }) {
   return (
-    <View
-      className="float"
-      style={[style, { flexShrink: 0, color: colors.n5 }]}
-    >
-      <Button type="bare" onClick={onFloat}>
+    <View className="float" style={[style, { flexShrink: 0 }]}>
+      <Button type="bare" onClick={onFloat} style={{ color: colors.n5 }}>
```
Comment (j-f1) [inline packages/desktop-client/src/components/sidebar.js:449]: Sorry one more time. The `color` style prop gets overridden on hover, which I had forgotten. The `color` prop overrides the default color.

```suggestion
      <Button type="bare" onClick={onFloat} color={colors.n5}>
```

Will open a PR shortly to fix a couple other instances of this 😬

Comment (MatissJanis) [reply packages/desktop-client/src/components/sidebar.js]: Thanks!

**Review by j-f1 — APPROVED** [review]

**Pull request merged**